### PR TITLE
Fix group assignment

### PR DESCRIPTION
**Pull request by NickLaMuro**
Changes the default group that the `dbreload` user is assigned to to be `dbreload` instead of opsworks.

This is because the `ssh_users` will remove any use that's default group is opsworks.  Since there really is no really win to having this group a part of the `opsworks` group, creating a group that users are able to join into if they want to share privileges seems like a resonable solution.

### CODE REVIEW

File: recipes/_user_base.rb [hunk 1]
```diff
@@ -1,8 +1,11 @@
 user = node[:opsworks_dbreload][:user]
 group = node[:opsworks_dbreload][:group]
-home_dir = "/home/#{user}" 
+home_dir = "/home/#{user}"
 
-group group
+group group do
```
Comment (bbergstrom) [inline recipes/_user_base.rb]: I think you can omit the group resource all together.  When you create a new user and don't specify a group, the OS will automatically provision a new group named after the user.

File: attributes/default.rb [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # User
 default[:opsworks_dbreload][:user] = "dbreload"
-default[:opsworks_dbreload][:group] = "opsworks"
+default[:opsworks_dbreload][:group] = "dbreload"
```
Comment (bbergstrom) [inline attributes/default.rb]: I think there is still benefit in keeping the group for directories/files created as opsworks as it makes it convenient for us human opsworks users.  I suppose if the files have the same permission for group as everyone than it doesn't much matter.

File: recipes/_user_base.rb (hunk 1 above)
Comment (NickLaMuro) [inline recipes/_user_base.rb]: Kept this here for things to work in `kitchen`.  Not terribly happy with the solution, but I guess it makes the cookbook more universal.

**Comment by bbergstrom** [comment]
:+1:

**Pull request merged**